runbook.tf

```diff
@@ -0,0 +1,53 @@
+
+############ automation account  + runbook #############
+resource "azurerm_automation_runbook" "vm-start-stop" {
+  name                    = "${var.product}-vm-status-change-${var.env}"
+  location                = var.location
+  resource_group_name     = var.resource_group_name
+  automation_account_name = var.automation_account_name
+  log_verbose             = var.env == "prod" ? "false" : "true"
+  log_progress            = "false"
+  description             = "This is a powershell runbook used to stop and start ${var.product} VMs"
+  runbook_type            = "PowerShell"
+  content                 = file("${path.module}/vm-start-stop.ps1")
+
+  tags = var.tags
+}
+
+################# automation schedule #################
+resource "azurerm_automation_schedule" "vm-start-stop" {
+  for_each = { for aa_acc_runbook in var.auto_acc_runbooks : aa_acc_runbook.name => aa_acc_runbook }
+
+  name                    = "${var.product}-schedule-${each.value.start_vm == true ? "start" : "stop"}-vm-${replace(each.value.run_time, ":", "-")}-${var.env}"
```

Comment:
can probably change this to `schedule-${each.value.start_vm == true ? "start" : "stop"}-vm-${replace(each.value.run_time, ":", "-")}`
as the other information will be inherit of what automation account it is in.

Reply:
I can see the point in that, I thinks it's easier for all resources to follow the naming convention.

I have updated to remove the env.